recipes/bvista/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "bvista" %}
+{% set version = "0.1.11" %}
+{% set python_min = "3.8" %}  # Minimum supported Python version on conda-forge
```

Comment:
Minimum supported version of python on conda-forge is 3.9, so you should remove this.